extractor/src/main/java/org/schabi/newpipe/extractor/stream/VideoStream.java

```diff
@@ -26,23 +26,18 @@
     public final String resolution;
     public final boolean isVideoOnly;
 
-
-    public VideoStream(String url, MediaFormat format, String resolution) {
-        this(url, format, resolution, false);
-    }
-
-    public VideoStream(String url, MediaFormat format, String resolution, boolean isVideoOnly) {
-        super(url, format);
-        this.resolution = resolution;
-        this.isVideoOnly = isVideoOnly;
+    public VideoStream(DeliveryFormat deliveryFormat, MediaFormat format, String resolution) {
+        this(null, deliveryFormat, format, resolution, false);
     }
 
-    public VideoStream(String url, String torrentUrl, MediaFormat format, String resolution) {
```

Comment:
You've replaced 4 `VideoStream`s by 3. Why doesn't this one have a replacement?

Reply:
Also, what is the OTF delivery format? Can't find any information online whatsoever.

Reply:
> You've replaced 4 VideoStreams by 3. Why doesn't this one have a replacement?

It was used only in one place I think, so I just removed it. It doesn't make a difference anyway.

> Also, what is the OTF delivery format? Can't find any information online whatsoever.

Yeah, I don't know exactly, didn't dig too much on it, but it seems to be some sort of in-house playback solution by YouTube,  it works pretty much like DASH.